src/+datools/+gaussian/EKF.m

```diff
@@ -0,0 +1,78 @@
+classdef EKF < datools.gaussian.GaussianFilter
```

Comment:
Could we call the +gaussian folder the +gaussianfilter? 

On a completely different note, maybe a different PR. can we have +filters and +smoothers, where +filters has +gaussianfilters, +statistical, and +variational?

Reply:
I like the +filter idea, but I won't do "filter.gaussianfilter" that is redundant